servers/rendering/rasterizer_rd/shaders/particles.glsl

```diff
@@ -158,11 +205,9 @@ bool emit_particle(mat4 p_xform, vec3 p_velocity, vec4 p_color, vec4 p_custom, u
 	}
 
 	barrier();
+=======
+>>>>>>> Implement GPU Particle Collisions
```

Comment:
Unresolved rebase changes here.

Reply:
With `<<<<<<< HEAD` ... `>>>>>>> Implement GPU Particle Collisions` part removed, and `override` keywords added in the `scene/3d/gpu_particles_collision_3d.h` it builds and starts without any errors on macOS.